lua/nvim-tree/utils.lua

```diff
@@ -210,6 +210,13 @@ function M.canonical_path(path)
   return path
 end
 
+-- Escapes special characters in string if windows else returns unmodified string.
+-- @param string string
+-- @return string
+function M.escape_windows_special_chars(string)
```

Comment:
Let's be a little more generic:

`escape_windows_special_chars(path)`

as string is the name of the string library.

Reply:
Updated.